config/initializers/secure_headers.rb

```diff
@@ -1,10 +1,40 @@
 SecureHeaders::Configuration.default do |config|
 
-  # Unblock PDF downloading for student report
+  # Unblock PDF downloading for student report and for IEP-at-a-glance
   config.x_download_options = nil
+  
+  if EnvironmentVariable.is_true('ENABLE_CSP')
+    config.csp = SecureHeaders::OPT_OUT # don't enforce yet
```

Comment:
@alexsoble So this wasn't working locally because the default behavior of `secure_headers` is to add CSP unless you opt-out.  So this was adding both a CSP policy (the default) and then our own "report only" header too.

This updates it to opt-out of CSP defaults and only add a report-only header to start, and I verified this locally.  Good catch! 👍